chains/optimism-sepolia-testnet.json

```diff
@@ -7,7 +7,7 @@
       "key": {
         "required": true
       },
-      "url": "https://api-sepolia-optimistic.etherscan.io/api"
+      "url": "https://api-sepolia-optimism.etherscan.io/api"
```

Comment:
I tracked down where this came from- the etherscan optimism sepolia docs landing page has this "optimistic" typo:

https://docs.optimism.etherscan.io/v/optimism-sepolia-etherscan